sweagent/environment/swe_env.py

```diff
@@ -182,13 +184,12 @@ def __init__(self, args: EnvironmentArguments):
         self.image_name = args.image_name
         self.container_obj: docker.models.containers.Container | None = None
         self.container: subprocess.Popen | None = None
-        self._reset_container()
+        # self._reset_container()
```

Comment:
**[swe-agent bug]** Initialize the default container, no matter if we need it or not.